irina/ex.js

```diff
@@ -0,0 +1,76 @@
+// # Function Scope
+
+// #### 1. Print Exponential Values
+// * Write a function that accepts two numbers and validate that they are numbers. 
+
+function exponetial1(num1,expo1) {
+console.log(typeof (num1 ** expo1))
+} 
+exponetial1(3,5) //number
+
+// After that, the function should print _y_ exponential values starting from _x_.
+
+
+function exponetial2(x,y) {
```

Comment:
will be great if you validate x and y before for loop in exponetial2 function

Reply:
done